UNIXFS.md

```diff
@@ -23,7 +23,8 @@ Draft work and discussion on a specification for the upcoming version 2 of the U
 - [Exporting](#exporting)
 - [Design decision rationale](#design-decision-rationale)
 	- [Metadata](#metadata-1)
-		- [Separate Metadata node](#separate-metadata-node)
+		- [Separate 
+		node](#separate-metadata-node)
```

Comment:
```suggestion
		- [Separate node](#separate-metadata-node)
```

Reply:
Fixed, I've managed to make a typo in a typo fix patch.